packages/fiori/src/themes/base/SearchField-parameters.css

```diff
@@ -1,7 +1,9 @@
 :root {
 	--_ui5_search_icon_border_radius: 1.125rem;
+	--_ui5_button_focused_border_radius: 1.125rem;
```

Comment:
Don't overwrite other component css vars (we do it only when we have no choice). Instead create ui5 search button specific var and use it in your css.